virttest/migration.py

```diff
@@ -37,7 +38,7 @@ def __init__(self):
         # The CmdResult returned from virsh migrate command
         self.ret = None
 
-    def post_migration_check(self, vms, params, uptime, uri=None):
+    def post_migration_check(self, vms, params, uptime=None, uri=None):
```

Comment:
Please update the docstring with more details about the usage.

Reply:
updated.